assets/js/editor.js

```diff
@@ -0,0 +1,10 @@
+import { FontLoader } from './tools';
```

Comment:
We probably don't want to load the theme fonts into the editor/CMS - perhaps we can add a placeholder or just console log for the editor file to be generated for now?